src/pytorch_lightning/core/datamodule.py

```diff
@@ -124,41 +131,41 @@ def from_datasets(
 
         """
 
-        def dataloader(ds: Dataset, shuffle: bool = False) -> DataLoader:
+        def dataloader(ds: Any, shuffle: bool = False) -> DataLoader:
```

Comment:
I don't think `Any` is appropriate here

Reply:
updated in https://github.com/Lightning-AI/lightning/pull/13693/commits/2f6e162ab7e399e38b61d4ff57e2982fe29eceb7